docs/api/feedback.md

```diff
@@ -0,0 +1,72 @@
+# Speech Feedback
+
+## Create a speech feedback
+
+When a user has performed a speech recording and received feedback on it, the
+feedback can be stored using this call.
+The feedback is created for the current user.
+
+### URL
+
+```http
+POST /challenges/speech/:challenge/feedbacks HTTP/1.1
```

Comment:
don't know about the 's' in 'feedbacks'. It should be just 'feedback': https://www.quora.com/What-is-the-plural-of-feedback-Can-you-ever-use-feedbacks

Reply:
Good catch, fixed